scripts/LedSignMock.py

```diff
@@ -0,0 +1,52 @@
+import curses
+import argparse 
+from server import led_sign_servicer
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+class LedSignMock(data):
+
+    menu = ['   ','SCE LED Sign Visualization ', ' ','Current Text: ', 'Current Brightness: ',' ',
+    'Current Scroll Speed:', 'Current Background Color: ', 'Current Font Color:',
+    'Current Border Color: ' ]
+    ESCKEY = 27
+
+    def _init_(self):
+        print('LED Mock running.')
+
+    def update_sign_visual(self, request):
+        self.sign_data["text"] = request.text
+        self.sign_data["brightness"] = request.brightness
+        self.sign_data["scroll-speed"] = request.scroll_speed
+        self.sign_data["background-color"] = request.background_color
+        self.sign_data["font-color"] = request.text_color
+        self.sign_data["border-color"] = request.border_color
+
+    def print_menu(self, stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        height, width = stdscr.getmaxyx()
+        for idx, row in enumerate(menu): #iterates over different menu options and aligns it
+            x = width//3 - len(menu) 
+            #calculates the location of the words (division changed by # of things in menu)
+            y = height//3 - len(menu)//3 + idx
+            stdscr.addstr(y, x, row)
+        stdscr.refresh()
+
+    def creates_display(self, stdscr):
+        curses.curs_set(0)
+        current_row = 0
+        print_menu(stdscr, current_row)
+        checkKey = stdscr.getch()
+        while checkKey != self.ESCKEY:
+            checkKey = stdscr.getch()
+        curses.endwin()
+        exit()
+        print_menu(stdscr, current_row)
+mock = LedSignMock()
```

Comment:
this code is outside the class and will automatically run as soon as you import the class. if you dont need it anymore consider removing it